file.sample.toml

```diff
@@ -0,0 +1,11 @@
+[tls.options.default]

```

Comment:
NIT: maybe we want to rename this file to `allowedCipher.toml`?

Reply:
`file.toml` signifies that this is a configuration for traefik [file provider](https://docs.traefik.io/v2.0/providers/file/). Unfortunately as of now this is the only way to specify TLS options. But depending on your requirements this file can have your additional dynamic configuration, so I'm not sure that specific name makes sense in this case. If you have strong feelings about this, I can rename it.

Reply:
Oh okay. That makes sense!  I think we are good for now with this file name. ;)